validator/remote/src/test/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandlerTest.java

```diff
@@ -395,11 +384,11 @@ public void getAttestationDuties_WhenFound_ReturnsDuties() {
             validatorCommitteeIndex,
             UInt64.ZERO);
 
-    when(apiClient.getAttestationDuties(ONE, IntList.of(validatorIndex)))
+    when(typeDefClient.postAttesterDuties(ONE, IntList.of(validatorIndex)))
         .thenReturn(
             Optional.of(
-                new PostAttesterDutiesResponse(
-                    dataStructureUtil.randomBytes32(), List.of(schemaValidatorDuties), false)));
+                new AttesterDuties(
+                    false, dataStructureUtil.randomBytes32(), List.of(expectedValidatorDuties))));
```

Comment:
do we actually have a test somewhere that tests this endpoint? this PR kind of suggests we don't at all -  this one is mocked and no other test changes...

Reply:
just adding to this, i think we're planning on filling this out, so im currently waiting until they're done or we chat about it.